src/decrypter.js

```diff
@@ -52,114 +50,138 @@ ntoh = function(word) {
 };
 
 /**
- * Schedule out an AES key for both encryption and decryption. This
- * is a low-level class. Use a cipher mode to do bulk encryption.
+ * Expand the S-box tables.
  *
- * @constructor
- * @param key {Array} The key as an array of 4, 6 or 8 words.
+ * @private
  */
-AES = function (key) {
-  this._precompute();
+const precompute = function() {
+  let _tables = [[[], [], [], [], []], [[], [], [], [], []]];
```

Comment:
well we made the function local to the module in this case, and we don't export it so essentially it is private now. My gut feeling on the prefix with _ rather than postfix _ was probably from a copy paste.